[package] gdal/3.1.0: does not compile with option with_podofo=True

### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **gdal/3.1.0**
  * Operating System+version: **Windows10**
  * Compiler+version: **msvc 14.0**
  * Docker image: **(none)**
  * Conan version: **conan 1.45.0**
  * Python version: **Python 3.9.0**


### Conan profile (output of `conan profile show default` or `conan profile show <profile>` if custom profile is in use)
```
[settings]
os=Windows
os_build=Windows
arch=x86_64
arch_build=x86_64
compiler=Visual Studio
compiler.version=15
build_type=Release
[options]
[conf]
[build_requires]
[env]
```


### Steps to reproduce (Include if Applicable)
Attempt to build the gdal/3.1.0 package with podofo:
```
PS E:\ws\conan-center-index\recipes\gdal\all> conan create . "gdal/3.1.0@" -o "gdal:with_podofo=True" --require-override "openssl/1.1.1m"
```


### Logs (Include/Attach if Applicable)
<details><summary>Click to expand log</summary>

```
PS E:\ws\conan-center-index\recipes\gdal\all> conan create . "gdal/3.1.0@" -o "gdal:with_podofo=True" --require-override "openssl/1.1.1m"
Exporting package recipe
gdal/3.1.0 exports: File 'conandata.yml' found. Exporting it...
gdal/3.1.0 exports: Copied 1 '.yml' file: conandata.yml
gdal/3.1.0: Calling export_sources()
gdal/3.1.0 export_sources() method: Copied 8 '.patch' files
gdal/3.1.0: The stored package has not changed
gdal/3.1.0: Using the exported files summary hash as the recipe revision: 52427cfa4cb1a4eebfde2272b4f13712 
gdal/3.1.0: Exported revision: 52427cfa4cb1a4eebfde2272b4f13712
Configuration:
[settings]
arch=x86_64
arch_build=x86_64
build_type=Release
compiler=Visual Studio
compiler.runtime=MD
compiler.version=15
os=Windows
os_build=Windows
[options]
gdal:with_podofo=True
[build_requires]
[env]

WARN: libgeotiff/1.7.0: requirement proj/8.2.1 overridden by gdal/3.1.0 to proj/8.1.1 
WARN: proj/8.1.1: requirement sqlite3/3.37.1 overridden by libgeotiff/1.7.0 to sqlite3/3.36.0 
WARN: libcurl/7.80.0: requirement openssl/1.1.1l overridden by proj/8.1.1 to openssl/1.1.1m 
gdal/3.1.0: Forced build from source
gdal/3.1.0 (test package): Installing package
Requirements
    brotli/1.0.9 from 'conancenter' - Cache
    bzip2/1.0.8 from 'conancenter' - Cache
    flatbuffers/2.0.0 from 'conancenter' - Cache
    freetype/2.11.1 from 'conancenter' - Cache
    gdal/3.1.0 from 'conancenter' - Cache
    geos/3.9.1 from 'conancenter' - Cache
    getopt-for-visual-studio/20200201 from 'conancenter' - Cache
    giflib/5.2.1 from 'conancenter' - Cache
    jbig/20160605 from 'conancenter' - Cache
    json-c/0.15 from 'conancenter' - Cache
    libcurl/7.80.0 from 'conancenter' - Cache
    libdeflate/1.9 from 'conancenter' - Cache
    libgeotiff/1.7.0 from 'conancenter' - Cache
    libiconv/1.16 from 'conancenter' - Cache
    libidn/1.36 from 'conancenter' - Cache
    libjpeg/9d from 'conancenter' - Cache
    libpng/1.6.37 from 'conancenter' - Cache
    libtiff/4.3.0 from 'conancenter' - Cache
    libwebp/1.2.2 from 'conancenter' - Cache
    nlohmann_json/3.10.4 from 'conancenter' - Cache
    openssl/1.1.1m from 'conancenter' - Cache
    podofo/0.9.7 from 'conancenter' - Cache
    proj/8.1.1 from 'conancenter' - Cache
    qhull/8.0.1 from 'conancenter' - Cache
    sqlite3/3.36.0 from 'conancenter' - Cache
    xz_utils/5.2.5 from 'conancenter' - Cache
    zlib/1.2.11 from 'conancenter' - Cache
    zstd/1.5.2 from 'conancenter' - Cache
Packages
    brotli/1.0.9:6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7 - Cache
    bzip2/1.0.8:5be2b7a2110ec8acdbf9a1cea9de5d60747edb34 - Cache
    flatbuffers/2.0.0:498affe1085dfb9382811207312ceadb108f5c12 - Cache
    freetype/2.11.1:48d0120dc306b44fcc074dbed3143a1cab49596c - Download
    gdal/3.1.0:46eec37cb1600ad2bee839602c4c2a6c58172e81 - Build
    geos/3.9.1:644a53c8938d7eb05995d8e50a5e506cae8ca2a5 - Cache
    getopt-for-visual-studio/20200201:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Cache
    giflib/5.2.1:d17d21c54da8f5460915a6e1b013121721c208ec - Cache
    jbig/20160605:1ed3f33597709b1f18c5a6ba2afcc72d12ddb2fc - Cache
    json-c/0.15:6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7 - Cache
    libcurl/7.80.0:9312e7a1647e82b66cda6b0bdfc4357128fdf1a7 - Cache
    libdeflate/1.9:6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7 - Cache
    libgeotiff/1.7.0:2c42f51484d2aa2a3c7ff8845702f4f300966d51 - Cache
    libiconv/1.16:6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7 - Cache
    libidn/1.36:d655199abcba725cc42fc68ca72324813a18f875 - Cache
    libjpeg/9d:6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7 - Cache
    libpng/1.6.37:e19a6a0686b274133d01476019f315a1355835d4 - Cache
    libtiff/4.3.0:d16796e2f9bea341f7a748858b34dc9ffb971a1c - Cache
    libwebp/1.2.2:36cedbf3473b284f710724d0897d2340d94bb47e - Cache
    nlohmann_json/3.10.4:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9 - Cache
    openssl/1.1.1m:6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7 - Download
    podofo/0.9.7:71f1270a57e5d0fa70c5d63e2a2b3526b54c5c0d - Download
    proj/8.1.1:d53c3f051edc83f4f362ae85f2ce2cda75af1998 - Cache
    qhull/8.0.1:6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7 - Cache
    sqlite3/3.36.0:31385508ee24f8d725d887c9dd6454318c15c5ac - Cache
    xz_utils/5.2.5:6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7 - Cache
    zlib/1.2.11:6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7 - Cache
    zstd/1.5.2:addbd53f27d0f386e44455f3159b5c679fbdb882 - Cache

Installing (downloading, building) binaries...
openssl/1.1.1m: Retrieving package 6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7 from remote 'conancenter' 
Downloading conanmanifest.txt completed [6.46k]
Downloading conaninfo.txt completed [2.49k]
Downloading conan_package.tgz completed [9025.84k]
Decompressing conan_package.tgz completed [0.00k]
openssl/1.1.1m: Package installed 6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7
openssl/1.1.1m: Downloaded package revision 0
freetype/2.11.1: Retrieving package 48d0120dc306b44fcc074dbed3143a1cab49596c from remote 'conancenter' 
Downloading conanmanifest.txt completed [4.43k]
Downloading conaninfo.txt completed [1.07k]
Downloading conan_package.tgz completed [844.05k]
Decompressing conan_package.tgz completed [0.00k]
freetype/2.11.1: Package installed 48d0120dc306b44fcc074dbed3143a1cab49596c
freetype/2.11.1: Downloaded package revision 0
podofo/0.9.7: Retrieving package 71f1270a57e5d0fa70c5d63e2a2b3526b54c5c0d from remote 'conancenter' 
Downloading conanmanifest.txt completed [7.49k]
Downloading conaninfo.txt completed [4.01k]
Downloading conan_package.tgz completed [2298.11k]
Decompressing conan_package.tgz completed [0.00k]
podofo/0.9.7: Package installed 71f1270a57e5d0fa70c5d63e2a2b3526b54c5c0d
podofo/0.9.7: Downloaded package revision 0
brotli/1.0.9: Already installed!
bzip2/1.0.8: Already installed!
bzip2/1.0.8: Appending PATH environment variable: Q:\.conan\bzip2\1.0.8\_\_\package\5be2b7a2110ec8acdbf9a1cea9de5d60747edb34\bin
flatbuffers/2.0.0: Already installed!
geos/3.9.1: Already installed!
getopt-for-visual-studio/20200201: Already installed!
jbig/20160605: Already installed!
jbig/20160605: Appending PATH environment variable: Q:\.conan\jbig\20160605\_\_\package\1ed3f33597709b1f18c5a6ba2afcc72d12ddb2fc\bin
json-c/0.15: Already installed!
libdeflate/1.9: Already installed!
libiconv/1.16: Already installed!
libiconv/1.16: Appending PATH environment var: Q:\.conan\libiconv\1.16\_\_\package\6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7\bin
libjpeg/9d: Already installed!
libwebp/1.2.2: Already installed!
nlohmann_json/3.10.4: Already installed!
qhull/8.0.1: Already installed!
qhull/8.0.1: Appending PATH environment variable: Q:\.conan\qhull\8.0.1\_\_\package\6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7\bin
sqlite3/3.36.0: Already installed!
sqlite3/3.36.0: Appending PATH env var with : Q:\.conan\sqlite3\3.36.0\_\_\package\31385508ee24f8d725d887c9dd6454318c15c5ac\bin
xz_utils/5.2.5: Already installed!
zlib/1.2.11: Already installed!
zstd/1.5.2: Already installed!
giflib/5.2.1: Already installed!
giflib/5.2.1: Appending PATH environment variable: Q:\.conan\giflib\5.2.1\_\_\package\d17d21c54da8f5460915a6e1b013121721c208ec\bin
libcurl/7.80.0: Already installed!
libidn/1.36: Already installed!
libidn/1.36: Appending PATH environment variable: Q:\.conan\libidn\1.36\_\_\package\d655199abcba725cc42fc68ca72324813a18f875\bin
libpng/1.6.37: Already installed!
libtiff/4.3.0: Already installed!
proj/8.1.1: Already installed!
proj/8.1.1: Prepending to PROJ_LIB environment variable: Q:\.conan\proj\8.1.1\_\_\package\d53c3f051edc83f4f362ae85f2ce2cda75af1998\res
proj/8.1.1: Appending PATH environment variable: Q:\.conan\proj\8.1.1\_\_\package\d53c3f051edc83f4f362ae85f2ce2cda75af1998\bin
libgeotiff/1.7.0: Already installed!
gdal/3.1.0: Configuring sources in Q:\.conan\gdal\3.1.0\_\_\source
Downloading gdal-3.1.0.tar.gz completed [18355.86k]

gdal/3.1.0: Copying sources to build folder
gdal/3.1.0: Building your package in Q:\.conan\gdal\3.1.0\_\_\build\46eec37cb1600ad2bee839602c4c2a6c58172e81
gdal/3.1.0: Generator pkg_config created json-c.pc
gdal/3.1.0: Generator pkg_config created libgeotiff.pc
gdal/3.1.0: Generator pkg_config created libflatbuffers.pc
gdal/3.1.0: Generator pkg_config created flatbuffers.pc
gdal/3.1.0: Generator pkg_config created giflib.pc
gdal/3.1.0: Generator pkg_config created geos_cxx_flags.pc
gdal/3.1.0: Generator pkg_config created geos_cpp.pc
gdal/3.1.0: Generator pkg_config created geos_c.pc
gdal/3.1.0: Generator pkg_config created geos.pc
gdal/3.1.0: Generator pkg_config created qhullstatic.pc
gdal/3.1.0: Generator pkg_config created libpodofo-0.pc
gdal/3.1.0: Generator pkg_config created proj.pc
gdal/3.1.0: Generator pkg_config created getopt-for-visual-studio.pc
gdal/3.1.0: Generator pkg_config created freetype2.pc
gdal/3.1.0: Generator pkg_config created libidn.pc
gdal/3.1.0: Generator pkg_config created libtiff-4.pc
gdal/3.1.0: Generator pkg_config created nlohmann_json.pc
gdal/3.1.0: Generator pkg_config created sqlite3.pc
gdal/3.1.0: Generator pkg_config created libcurl.pc
gdal/3.1.0: Generator pkg_config created libiconv.pc
gdal/3.1.0: Generator pkg_config created libpng.pc
gdal/3.1.0: Generator pkg_config created bzip2.pc
gdal/3.1.0: Generator pkg_config created libbrotlicommon.pc
gdal/3.1.0: Generator pkg_config created libbrotlienc.pc
gdal/3.1.0: Generator pkg_config created libbrotlidec.pc
gdal/3.1.0: Generator pkg_config created brotli.pc
gdal/3.1.0: Generator pkg_config created zlib.pc
gdal/3.1.0: Generator pkg_config created libdeflate.pc
gdal/3.1.0: Generator pkg_config created liblzma.pc
gdal/3.1.0: Generator pkg_config created libjpeg.pc
gdal/3.1.0: Generator pkg_config created jbig.pc
gdal/3.1.0: Generator pkg_config created libzstd.pc
gdal/3.1.0: Generator pkg_config created libwebp.pc
gdal/3.1.0: Generator pkg_config created libwebpmux.pc
gdal/3.1.0: Generator pkg_config created libwebpdemux.pc
gdal/3.1.0: Generator pkg_config created libwebpdecoder.pc
gdal/3.1.0: Generator pkg_config created libcrypto.pc
gdal/3.1.0: Generator pkg_config created libssl.pc
gdal/3.1.0: Generator pkg_config created openssl.pc
gdal/3.1.0: Aggregating env generators
gdal/3.1.0: Calling build()
**********************************************************************
** Visual Studio 2017 Developer Command Prompt v15.9.38
** Copyright (c) 2017 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64'

Microsoft (R) Program Maintenance Utility Version 14.16.27045.0
Copyright (C) Microsoft Corporation.  All rights reserved.

        cd port
        nmake /nologo /f makefile.vc
        call prev_dllbuild.bat

```
(cut)
```
Q:\.conan\gdal\3.1.0\_\_\build\46eec37cb1600ad2bee839602c4c2a6c58172e81\source_subfolder\frmts>cd pdf   && nmake /nologo /f makefile.vc   && cd ..   || exit 1
        cl   /nologo /MP%NUMBER_OF_PROCESSORS% /EHsc /FC /D_CRT_SECURE_NO_DEPRECATE /D_CRT_NONSTDC_NO_DEPRECATE /DNDEBUG /W4 /wd4127 /wd4251 /wd4275 /wd4786 /wd4100 /wd4245 /wd4206 /wd4351 /wd4611 
 /DHAVE_SSE_AT_COMPILE_TIME  -I..\..\port -I..\..\ogr -I..\..\gcore  -I..\..\alg -I..\..\ogr\ogrsf_frmts  -I..\..\gnm -I..\..\gnm\gnm_frmts -I..\..\apps  -I..\vrt -I..\mem -I..\..\ogr\ogrsf_frmts\mem   -DHAVE_PODOFO   -DGNM_ENABLED  -DGDAL_COMPILATION -DNOMINMAX /c pdfdataset.cpp pdfio.cpp pdfobject.cpp pdfcreatecopy.cpp ogrpdflayer.cpp pdfwritabledataset.cpp pdfreadvectors.cpp pdfcreatefromcomposition.cpp
pdfdataset.cpp
pdfio.cpp
pdfobject.cpp
pdfcreatecopy.cpp
ogrpdflayer.cpp
pdfwritabledataset.cpp
pdfreadvectors.cpp
pdfcreatefromcomposition.cpp
q:\.conan\gdal\3.1.0\_\_\build\46eec37cb1600ad2bee839602c4c2a6c58172e81\source_subfolder\frmts\pdf\pdfsdk_headers.h(106): fatal error C1083: Cannot open include file: 'podofo.h': No such file or directory (compiling source file pdfwritabledataset.cpp)
q:\.conan\gdal\3.1.0\_\_\build\46eec37cb1600ad2bee839602c4c2a6c58172e81\source_subfolder\frmts\pdf\pdfsdk_headers.h(106): fatal error C1083: Cannot open include file: 'podofo.h': No such file or directory (compiling source file pdfobject.cpp)
q:\.conan\gdal\3.1.0\_\_\build\46eec37cb1600ad2bee839602c4c2a6c58172e81\source_subfolder\frmts\pdf\pdfsdk_headers.h(106): fatal error C1083: Cannot open include file: 'podofo.h': No such file or directory (compiling source file pdfio.cpp)
q:\.conan\gdal\3.1.0\_\_\build\46eec37cb1600ad2bee839602c4c2a6c58172e81\source_subfolder\frmts\pdf\pdfsdk_headers.h(106): fatal error C1083: Cannot open include file: 'podofo.h': No such file or directory (compiling source file pdfdataset.cpp)
q:\.conan\gdal\3.1.0\_\_\build\46eec37cb1600ad2bee839602c4c2a6c58172e81\source_subfolder\frmts\pdf\pdfsdk_headers.h(106): fatal error C1083: Cannot open include file: 'podofo.h': No such file or directory (compiling source file pdfreadvectors.cpp)
q:\.conan\gdal\3.1.0\_\_\build\46eec37cb1600ad2bee839602c4c2a6c58172e81\source_subfolder\frmts\pdf\pdfsdk_headers.h(106): fatal error C1083: Cannot open include file: 'podofo.h': No such file or directory (compiling source file pdfcreatefromcomposition.cpp)
q:\.conan\gdal\3.1.0\_\_\build\46eec37cb1600ad2bee839602c4c2a6c58172e81\source_subfolder\frmts\pdf\pdfsdk_headers.h(106): fatal error C1083: Cannot open include file: 'podofo.h': No such file or directory (compiling source file ogrpdflayer.cpp)
q:\.conan\gdal\3.1.0\_\_\build\46eec37cb1600ad2bee839602c4c2a6c58172e81\source_subfolder\frmts\pdf\pdfsdk_headers.h(106): fatal error C1083: Cannot open include file: 'podofo.h': No such file or directory (compiling source file pdfcreatecopy.cpp)
NMAKE : fatal error U1077: 'cl' : return code '0x2'
Stop.
NMAKE : fatal error U1077: 'for' : return code '0x1'
Stop.
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\VC\Tools\MSVC\14.16.27023\bin\HostX64\x64\nmake.EXE"' : return code '0x2'
Stop.
gdal/3.1.0:
gdal/3.1.0: ERROR: Package '46eec37cb1600ad2bee839602c4c2a6c58172e81' build failed
gdal/3.1.0: WARN: Build folder Q:\.conan\gdal\3.1.0\_\_\build\46eec37cb1600ad2bee839602c4c2a6c58172e81
ERROR: gdal/3.1.0: Error in build() method, line 810
        self.run("nmake -f makefile.vc {}".format(" ".join(self._get_nmake_args())))
        ConanException: Error 2 while executing nmake -f makefile.vc GDAL_HOME="Q:\.conan\gdal\3.1.0\_\_\package\46eec37cb1600ad2bee839602c4c2a6c58172e81" DATADIR="Q:\.conan\gdal\3.1.0\_\_\package\46eec37cb1600ad2bee839602c4c2a6c58172e81\res\gdal" WIN64=1 DEBUG=0 DLLBUILD=0 PROJ_INCLUDE="-IQ:\.conan\proj\8.1.1\_\_\package\d53c3f051edc83f4f362ae85f2ce2cda75af1998\include" LIBICONV_INCLUDE="-IQ:\.conan\libiconv\1.16\_\_\package\6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7\include" LIBICONV_CFLAGS="-DICONV_CONST=" JPEG_EXTERNAL_LIB=1 JPEGDIR="Q:\.conan\libjpeg\9d\_\_\package\6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7\include" PNG_EXTERNAL_LIB=1 PNGDIR="Q:\.conan\libpng\1.6.37\_\_\package\e19a6a0686b274133d01476019f315a1355835d4\include" GIF_SETTING=EXTERNAL PCRASTER_SETTING=INTERNAL TIFF_INC="-IQ:\.conan\libtiff\4.3.0\_\_\package\d16796e2f9bea341f7a748858b34dc9ffb971a1c\include" GEOTIFF_INC="-IQ:\.conan\libgeotiff\1.7.0\_\_\package\2c42f51484d2aa2a3c7ff8845702f4f300966d51\include" SQLITE_INC="-IQ:\.conan\sqlite3\3.36.0\_\_\package\31385508ee24f8d725d887c9dd6454318c15c5ac\include" GEOS_CFLAGS="-IQ:\.conan\geos\3.9.1\_\_\package\644a53c8938d7eb05995d8e50a5e506cae8ca2a5\include -DHAVE_GEOS" ZLIB_EXTERNAL_LIB=1 ZLIB_INC="-IQ:\.conan\zlib\1.2.11\_\_\package\6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7\include" PODOFO_ENABLED=YES QHULL_SETTING=EXTERNAL
```

</details>

--- a/recipes/gdal/all/conandata.yml
+++ b/recipes/gdal/all/conandata.yml
@@ -21,6 +21,8 @@ patches:
       base_path: "source_subfolder"
     - patch_file: "patches/3.4.x/fix-include-xerces.patch"
       base_path: "source_subfolder"
+    - patch_file: "patches/3.4.x/fix-include-podofo.patch"
+      base_path: "source_subfolder"
   "3.3.3":
     - patch_file: "patches/3.3.x/fix-autotools-3.3.3.patch"
       base_path: "source_subfolder"
@@ -30,6 +32,8 @@ patches:
       base_path: "source_subfolder"
     - patch_file: "patches/3.3.x/fix-include-xerces.patch"
       base_path: "source_subfolder"
+    - patch_file: "patches/3.3.x/fix-include-podofo.patch"
+      base_path: "source_subfolder"
   "3.2.3":
     - patch_file: "patches/3.2.x/fix-autotools.patch"
       base_path: "source_subfolder"
@@ -38,6 +42,8 @@ patches:
     - patch_file: "patches/3.2.x/fix-include-jsonc.patch"
       base_path: "source_subfolder"
     - patch_file: "patches/3.2.x/fix-include-xerces.patch"
+      base_path: "source_subfolder"
+    - patch_file: "patches/3.2.x/fix-include-podofo.patch"
       base_path: "source_subfolder"
   "3.1.4":
     - patch_file: "patches/3.1.x/fix-autotools-3.1.4.patch"
@@ -51,4 +57,6 @@ patches:
     - patch_file: "patches/3.1.x/fix-include-jsonc.patch"
       base_path: "source_subfolder"
     - patch_file: "patches/3.1.x/fix-include-xerces-3.1.2.patch"
+      base_path: "source_subfolder"
+    - patch_file: "patches/3.1.x/fix-include-podofo.patch"
       base_path: "source_subfolder"

--- a/recipes/gdal/all/patches/3.1.x/fix-include-podofo.patch
+++ b/recipes/gdal/all/patches/3.1.x/fix-include-podofo.patch
@@ -1,0 +1,11 @@
+--- a/frmts/pdf/pdfsdk_headers.h
++++ b/frmts/pdf/pdfsdk_headers.h
+@@ -103,7 +103,7 @@ typedef unsigned char Guchar;
+ #undef GetObject
+ #endif
+
+-#include "podofo.h"
++#include <podofo/podofo.h>
+ #endif // HAVE_PODOFO
+
+ #ifdef HAVE_PDFIUM

--- a/recipes/gdal/all/patches/3.2.x/fix-include-podofo.patch
+++ b/recipes/gdal/all/patches/3.2.x/fix-include-podofo.patch
@@ -1,0 +1,11 @@
+--- a/frmts/pdf/pdfsdk_headers.h
++++ b/frmts/pdf/pdfsdk_headers.h
+@@ -103,7 +103,7 @@ typedef unsigned char Guchar;
+ #undef GetObject
+ #endif
+
+-#include "podofo.h"
++#include <podofo/podofo.h>
+ #endif // HAVE_PODOFO
+
+ #ifdef HAVE_PDFIUM

--- a/recipes/gdal/all/patches/3.3.x/fix-include-podofo.patch
+++ b/recipes/gdal/all/patches/3.3.x/fix-include-podofo.patch
@@ -1,0 +1,11 @@
+--- a/frmts/pdf/pdfsdk_headers.h
++++ b/frmts/pdf/pdfsdk_headers.h
+@@ -103,7 +103,7 @@ typedef unsigned char Guchar;
+ #undef GetObject
+ #endif
+
+-#include "podofo.h"
++#include <podofo/podofo.h>
+ #endif // HAVE_PODOFO
+
+ #ifdef HAVE_PDFIUM

--- a/recipes/gdal/all/patches/3.4.x/fix-include-podofo.patch
+++ b/recipes/gdal/all/patches/3.4.x/fix-include-podofo.patch
@@ -1,0 +1,11 @@
+--- a/frmts/pdf/pdfsdk_headers.h
++++ b/frmts/pdf/pdfsdk_headers.h
+@@ -103,7 +103,7 @@ typedef unsigned char Guchar;
+ #undef GetObject
+ #endif
+
+-#include "podofo.h"
++#include <podofo/podofo.h>
+ #endif // HAVE_PODOFO
+
+ #ifdef HAVE_PDFIUM

